Modules/Loadable/Segmentations/EditorEffects/qSlicerSegmentEditorAbstractEffect.cxx

```diff
@@ -515,15 +515,33 @@ void qSlicerSegmentEditorAbstractEffect::cursorOff(qMRMLWidget* viewWidget)
   Q_D(qSlicerSegmentEditorAbstractEffect);
 
   d->SavedCursor = QCursor(viewWidget->cursor());
-  viewWidget->setCursor(QCursor(Qt::BlankCursor));
+  qMRMLSliceWidget* sliceWidget = qobject_cast<qMRMLSliceWidget*>(viewWidget);
+  qMRMLThreeDWidget* threeDWidget = qobject_cast<qMRMLThreeDWidget*>(viewWidget);
+  if (sliceWidget && sliceWidget->sliceView())
+    {
+    sliceWidget->sliceView()->setViewCursor(QCursor(Qt::BlankCursor));
```

Comment:
You may also want to look at the new method `setDefaultQVTKCursor`. See https://github.com/Slicer/VTK/commit/4ab10d5674090961d3fc48ac23319b3da97defec